### PR TITLE
added post and images

### PR DESCRIPTION
**Pull request by rafriki**

### CODE REVIEW

**Comment by f000** [comment]
congratz! :)

File: _posts/2016-01-25-charity-quiz-pilgrims-hospice.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+layout: 		post
+title:  		"Stour Chambers' Charity Quiz!"
+date:   		2016-01-25
+author: 		"John Parkinson"
```
Comment (lukefrake) [inline _posts/2016-01-25-charity-quiz-pilgrims-hospice.md]: this is author ID so yours is john_parkinson

**Comment by rafriki** [comment]
ty ty

File: _posts/2016-01-25-charity-quiz-pilgrims-hospice.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+authorPhoto:	"2aa226fd35a96b7dcf125f686d944245"
+categories: 	space between
+banner:			"/assets/images/blog/delegates.jpg"
+---
+
+We were invited to a charity quiz hosted by [Stour Chambers](http://www.stourchambers.co.uk) at The Wesgate Hall in Canterbury to raise money for The Pilgrims Hospice.
```
Comment (lukefrake) [inline _posts/2016-01-25-charity-quiz-pilgrims-hospice.md]: Can you say Canterbury, Kent

File: _posts/2016-01-25-charity-quiz-pilgrims-hospice.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+<img src="/assets/images/blog/delegates.jpg" alt="The delegates of the Stour Chambers' Charity Quiz" class="img img-vMargin">
+
+The quiz was organised by [Kate Kochnari](http://www.stourchambers.co.uk/barristers/kate-kochnari) and the marketing team there, and they asked whether we would donate any prizes for the raffle. In typical techy fashion, we got together and chose a few cool tech prizes:
```
Comment (lukefrake) [inline _posts/2016-01-25-charity-quiz-pilgrims-hospice.md]: nd the marketing team there, (don't need the second and) they

File: _posts/2016-01-25-charity-quiz-pilgrims-hospice.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+1. Amazon Fire 7 - This budget tablet is perfect for someone looking for an introduction into mobile computing. It comes with the Amazon App Store and a few pre-installed apps.
+2. Chromecast - Google's contribution to the TV dongle market, the Chromecast plugs into the back of your TV through HDMI and is controlled by a smartphone, laptop or tablet. It lets you stream from your laptop or computer and has some pre-installed apps like the BBC iPlayer and youtube.
+3. Amazon Fire TV stick - A competitior to the Chromecast, the 'fire stick' works in a similar way but comes with its own remote control. Netflix and iPlayer are installed out-of-the-box.
+4. Sailor Jerry's Limited Edition - We thought it wouldn't be a raffle donation without a good bottle of booze...
+
+The quiz included a music round, general knowledge, current affairs, aliasses,  cities of the world and a picture round. We got off to a flakey start with the aliasses round, but played our joker on the music round and gained a healthy lead. Mike Vigor from Big Blue Dev, a friend of ours at Space Between, has watched more films than we've written lines of code - he carried us through that round.
```
Comment (lukefrake) [inline _posts/2016-01-25-charity-quiz-pilgrims-hospice.md]: Double space after aliasses,

File: _posts/2016-01-25-charity-quiz-pilgrims-hospice.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+layout: 		post
+title:  		"Stour Chambers' Charity Quiz!"
+date:   		2016-01-25
+author_id: 		"john_parkinson"
+description:	"We were invited to a charity quiz hosted by Stour Chambers at The Wesgate Hall in Canterbury, Kent to raise money for The Pilgrims Hospice."
+categories: 	space between
+banner:			"/assets/images/blog/delegates.jpg"
+---
+
+We were invited to a charity quiz hosted by [Stour Chambers](http://www.stourchambers.co.uk) at The Wesgate Hall in Canterbury, Kent to raise money for The Pilgrims Hospice.
```
Comment (lukefrake) [inline _posts/2016-01-25-charity-quiz-pilgrims-hospice.md]: Westgate*

File: _posts/2016-01-25-charity-quiz-pilgrims-hospice.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+
+<img src="/assets/images/blog/delegates.jpg" alt="The delegates of the Stour Chambers' Charity Quiz" class="img img-vMargin">
+
+The quiz was organised by [Kate Kochnari](http://www.stourchambers.co.uk/barristers/kate-kochnari) with the marketing team there, and they asked whether we would donate any prizes for the raffle. In typical techy fashion, we got together and chose a few cool tech prizes:
+
+<img src="/assets/images/blog/prizes.jpg" alt="Space Between prize donations" class="img img-vMargin">
+
+1. Amazon Fire 7 - This budget tablet is perfect for someone looking for an introduction into mobile computing. It comes with the Amazon App Store and a few pre-installed apps.
+2. Chromecast - Google's contribution to the TV dongle market, the Chromecast plugs into the back of your TV through HDMI and is controlled by a smartphone, laptop or tablet. It lets you stream from your laptop or computer and has some pre-installed apps like the BBC iPlayer and youtube.
+3. Amazon Fire TV stick - A competitior to the Chromecast, the 'fire stick' works in a similar way but comes with its own remote control. Netflix and iPlayer are installed out-of-the-box.
```
Comment (lukefrake) [inline _posts/2016-01-25-charity-quiz-pilgrims-hospice.md]: competitor*

File: _posts/2016-01-25-charity-quiz-pilgrims-hospice.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+4. Sailor Jerry's Limited Edition - We thought it wouldn't be a raffle donation without a good bottle of booze...
+
+The quiz included a music round, general knowledge, current affairs, aliasses, cities of the world and a picture round. We got off to a flakey start with the aliasses round, but played our joker on the music round and gained a healthy lead. Mike Vigor from Big Blue Dev, a friend of ours at Space Between, has watched more films than we've written lines of code - he carried us through that round.
```
Comment (lukefrake) [inline _posts/2016-01-25-charity-quiz-pilgrims-hospice.md]: aliasses = aliases

File: _posts/2016-01-25-charity-quiz-pilgrims-hospice.md [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+
+We didn't expect to win and we had clearly all gone to have some fun as we quickly got a round of drinks in after the demorilizing aliasses round. As the night went on, we noticed that not only were we maintaining a healthy lead on the points, we were also storming ahead of everyone else in how much we were drinking!
```
Comment (lukefrake) [inline _posts/2016-01-25-charity-quiz-pilgrims-hospice.md]: demoralizing aliases*

**Comment by lukefrake** [comment]
:+1: looks good, if you can find a better banner image that would be great!

**Pull request merged**